### ISSUE TITLE
How to control lights' level and how to sync switches' status with ST app

### ISSUE THREAD

**Issue by casper-gh** [jnewland/SmartThings#7]
Thanks for the app, working great with homekit, but I have some questions, not sure if the features isn't implemented or am I missing something:
- Besides switching on/off the lights, I cannot control the level.
- The status on the switches aren't synced with ST app, for example, if I turn off the light from ST iOS app, the status in homekit isn't updated. However vice versa works.

**Comment by nzhome** [comment]
yeah, its not intuitive.  anyone know how to control the level? is it another ST smart app or something?
